src/styles/Global.ts

```diff
@@ -1,7 +1,6 @@
 import { createGlobalStyle } from "styled-components";
 
 export const GlobalStyles = createGlobalStyle`
-@import url('https://fonts.googleapis.com/css2?family=Maven+Pro&family=Ubuntu:wght@300&display=swap');
```

Comment:
to powinno zostać usunięte? tylko pytam

Reply:
W najnowszych zmianach, które pobrałem z main już tego tam nie widzę, więc chyba tak